config/core/300-resources/service.yaml

```diff
@@ -662,7 +662,7 @@ spec:
                                   description: |
                                     As specified in Kubernetes core/v1.Probe.
                                   default: 300
-                                  minimum: 0
+                                  minimum: 1
```

Comment:
I think it's complaining about the default rather than the minimum. The default should be 0 for Knative too. We have some specialcased behavior for when periodSeconds is set to 0. The tests are failing in our webhook validation.

Reply:
thanks, I'll update that